pkg/bundle/sync.go

```diff
@@ -244,12 +252,10 @@ func certAlias(derData []byte, friendlyName string) string {
 	return certHash[:8] + "|" + friendlyName
 }
 
-type pkcs12Encoder struct {
-	password string
-}
+func (e pkcs12Encoder) encode() ([]byte, error) {
+	var encoder = pkcs12.Modern2023 // user modern PKCS#12 encoder
```

Comment:
```suggestion
	var encoder = pkcs12.LegacyDES
```

When working on https://github.com/cert-manager/cert-manager/pull/6515 I realize that we prefer compatibility over creating a more secure trust store. PKCS12 and JKS are insecure anyway and should be protected by other means than the password.

Reply:
I took you remark into account in a new PR #244 . I decided to re-create it due to mess of remarks and uncertain decisions.